content/en/_index.md

```diff
@@ -71,7 +71,7 @@ cascade:
   <div class="col-md-6">
     <div class="card">
       <div class="card-content">
-  <h3><b class="title-color">Try Our DAST Scanner</b></h3>        
+<h3><b class="gradient-text">Try Our DAST Scanner</b></h3>
```

Comment:
code style: you might consider indenting the `<h3>` to be indented 2 spaces farther than the parent `<div>`.